.drone.yml

```diff
@@ -215,6 +215,19 @@ steps:
   - test-backend
   - test-frontend
 
+- name: release-next-npm-packages
+  image: grafana/build-container:1.2.30
+  commands:
+  - ./node_modules/.bin/lerna bootstrap
```

Comment:
nit: Not sure we need to run this again. I think the node_modules should already be setup in place.

Reply:
Yeah yarn workspaces should have sorted all that out for us. Will delete it. Thanks @mckn! 🙏